torchgeo/datamodules/eurosat.py

```diff
@@ -9,7 +9,7 @@
 import pytorch_lightning as pl
 import torch
 from torch.utils.data import DataLoader
-from torchvision.transforms import Compose, Normalize
```

Comment:
Removed `Normalize` as it is more explicit to subtract band means then divide by band stdevs.